### PR TITLE
Sunset `setUserRole` and `getUserRoles`

### LINKED ISSUE #304: Authority client methods need to be updated
## Description

`colonyClient.authority.setUserRole` is always unsuccessful. This is either outdated and needs to be removed from the docs or a bug that needs to be fixed.

I am able to successfully manage user roles with the following methods `colonyClient.setOwnerRole`, `colonyClient.setAdminRole`, and `colonyClient.removeAdminRole`.

### PR DESCRIPTION
**Pull request by JamesLefrere**
## Description

👋 

(This method is not supported by the contracts anymore).

Resolves #304 and #200

### CODE REVIEW

File: docs/_API_ColonyClient.md [hunk 1]
```diff
@@ -470,7 +470,7 @@ An instance of a `ContractResponse`
 
 ### `setRewardInverse.send({ rewardInverse }, options)`
 
-Set new colony recovery role. Can only be called by the founder role.
+Set the reward inverse to pay out from revenue. e.g. if the fee is 1% (or 0.01), set 100
```
Comment (JamesLefrere) [inline docs/_API_ColonyClient.md]: Seems like docgen wasn't called before 🤔

Comment (ryanchristo) [reply docs/_API_ColonyClient.md]: At least it's fixed now 👍

**Comment by JamesLefrere** [comment]
We can do that... perhaps we should talk to the network team about exposing `hasUserRole`, so we can do away with this class.

**Comment by JamesLefrere** [comment]
Let's see how https://github.com/JoinColony/colonyNetwork/pull/450 does.

**Comment by JamesLefrere** [comment]
I guess we can now ditch `AuthorityClient` altogether?

**Comment by JamesLefrere** [comment]
Closing in favour if #311

**Pull request closed without merging**